### ISSUE TITLE
Rendering error with Emacs

### ISSUE THREAD

**Issue by r3str0b3** [conformal/spectrwm#344]
Every time I use an incremental completion framework like IDO or helm to select an option from a list, a white rectangular bar appears at the bottom.

I thought it had something to do with the status bar being at the bottom, but the issue persists after I configured it to be displayed at the top.

It doesn't appear when the bar is enabled, but I like to have it disabled.

SpectrWM Config: https://pastebin.com/z94c9tSG

Any recommendations?

**Comment by r3str0b3** [comment]
This is issue does not appear when I use DWM, XMonad, or i3, so I doubt this is an emacs issue.

**Comment by dinoallo** [comment]
> Every time I use an incremental completion framework like IDO or helm to select an option from a list, a white rectangular bar appears at the bottom.
> 
> I thought it had something to do with the status bar being at the bottom, but the issue persists after I configured it to be displayed at the top.
> 
> It doesn't appear when the bar is enabled, but I like to have it disabled.
> 
> SpectrWM Config: https://pastebin.com/z94c9tSG
> 
> Any recommendations?

I had a similar problem with Emacs frame not filling the whole window. And I referred to the Emacs manual and found this:

> **User Option: frame-resize-pixelwise**
If this option is nil (the default), a frame’s text pixel size is usually rounded to a multiple of the current values of that frame’s frame-char-height and frame-char-width whenever the frame is resized. If this is non-nil, no rounding occurs, hence frame sizes can increase/decrease by one pixel. 
> ...
> **With some window managers you may have to set this to non-nil in order to make a frame appear truly maximized or full-screen.**
 
[Here](https://www.gnu.org/software/emacs/manual/html_node/elisp/Frame-Size.html) is the full description.
TL;DR set `frame-resize-pixelwise` to `t`  
```emacs-lisp
(setq frame-resize-pixelwise t)
```
And then my problem was solved. Hope this is relevant to your issue as well?
